hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -205,7 +215,84 @@ export const CheckedOutTables = () =>
 
 export const ReturnedTable = () => {
     const dispatch = useDispatch();
-    const orders = useSelector(returnedOrdersSelector);
+    const unsorted_orders = useSelector(returnedOrdersSelector);
+    const orders = unsorted_orders.slice().sort((order1, order2) => {
```

Comment:
Hey @liu-samuel  I notice you use similar code for both Participant and Admin side. Could create a helper function that does the sorting in `helpers.ts` or another file so it doesn't clutter `ItemTable.tsx`?